data/remnant outfits.txt

```diff
@@ -463,6 +463,19 @@ outfit "Tuning Rifle"
 	"unplunderable" 1
 	description "The Tuning Rifle uses sound waves to destabilize the weaker internal tissue of many races and liquefies them."
 
+outfit "Void Rifle"
+	category "Special"
+	licenses
+		Remnant
+	cost 1000000
+	thumbnail "outfit/void rifle"
+	"capture attack" 20
+	"capture defense" 20
+	"energy consumption" 0.017
+	"unplunderable" 1
+	description "The Remnant made a significant number of these weapons during the early years of their research on the void sprites, but have not been able to make any new ones since the Archon stopped their harvests. Without a supply of new tentacles to use, the Remnant have been unable to manufacture more. While durable, they do rarely last more than a decade, so even the best cared for of the old Void Rifles degraded into uselessness centuries ago. These highly effective weapons are credited with helping the Remnant safeguard their refuges from ground-based threats during their early years."
+	description "	When fired, the Void Rifle emits a form of radiation that has a strong paralyzing effect on the nervous system. With an area of effect that penetrates walls and other solid objects, it is an incredibly effective weapon."
```

Comment:
`a form of radiation` seems too much like vague-techno-babble to me. How about just `the Void Rifle emits a beam that has...` or even since it's biological in nature, `the Void Rifle emits a deadly neurotoxin that can result in death in seconds. It is simultaneously able to erode most substances, making it an incredibly effective weapon, but at the same time a dangerous one to yield.`.

Reply:
Changed `a form of radiation` to `a beam` as that stays the closest to the intended effect. It is an anti-personnel weapon that passes through walls and most solid objects without damaging them. (although, like radiation, walls and objects built out of materials designed to stop radiation would almost certainly be effective at blocking it)